css/dark.scss

```diff
@@ -32,12 +37,11 @@ $middle-bg: #222;
 
 // Other fancy stuff
 $corner-radius: 10px;
-$shadow-style: 5px 5px 3px #000000;
+$shadow-style: 5px 5px 3px $black;
 $inset-shadow-style: inset $shadow-style;
-$menu-border-style: 1px solid #ff0000;
-
-$code-border-style: solid 1px $white ;
+$menu-border-style: 1px solid $red;
 
+// DON'T CHANGE THIS
```

Comment:
It is for the user's benifit, iff it will work in the included files fine, then I will go ahead and move it into one, which do you sugest I put it in: layout, mixins or style?